### PR TITLE
Documentación par los contratos eventuales y el calculo del consumo

### PR DESCRIPTION
**Pull request by polsala**
- [x] Added documentation.
- [x] Added translations to Spanish.

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

File: docs/base/polizas.md [hunk 1]
```diff
@@ -321,6 +321,62 @@ flux d'aquesta.
...
+Dins d'aquests contractes en podem distingir dos tipus:
+
+* **Contracte eventual sense comptador**: aquests són contractes d'una duració
+màxima de dos mesos en els quals per llei no fa falta un comptador, el consum es
+calcularà a partir d'una fórmula (Potència contractada * dies de contracte * hores diaries).
+
+* **Contracte eventual amb comptador**: a diferencia del contracte anterior aquest
+sí que disposa de comptador per poder calcular el consum.
+
+## Càlcul del consum total per contractes eventuals sense comptador
```
Comment (ecarreras) [inline docs/base/polizas.md]: Better to use header 3 (###) and include this section inside of "Contractes de tipus eventual"

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_NEW_functionality_eventual_contracts_no_meters/

**Pull request merged**